src/main/scripts/src/app/shared/perspectives/documents/post-it/attribute-tree/attribute/attribute-tree-attribute.component.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *
+ * Copyright (C) since 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+
+import {Component} from '@angular/core';
+
+import {AttributeTreeComponent} from '../attribute-tree.component';
+
+@Component({
+  selector: 'attribute',
+  templateUrl: './attribute-tree-attribute.component.html',
```

Comment:
It's there to separate the attribute tree parent from it's children that look and might behave different.
(Where attribute-tree component handles how document attributes are displayed, such as delimiters between attributes, hover highlighting.. While attribute handles displaying attribute value, parsing and displaying any string, array, null, or object) :)